<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPort Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/res/logo.png</normaloff>:/res/logo.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openCloseButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>661</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="recvEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>671</width>
      <height>291</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="hexSendBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX Send</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="loopBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cycleBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="suffix">
     <string> ms</string>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="hexShowBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX Show</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clearBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wordWrapBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Word Wrap</string>
    </property>
   </widget>
   <widget class="QLed" name="led" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_ile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Send_File"/>
    <addaction name="action_Setup"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuF_ile"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Send_File">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/setup.png</normaloff>:/res/setup.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Send File</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>:/s/res/setup.png</normaloff>:/s/res/setup.png</iconset>
   </property>
   <property name="text">
    <string>Serial port setup</string>
   </property>
  </action>
  <action name="action_Setup">
   <property name="text">
    <string>setup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
